cli/pkg/kctrl/cmd/package/release/release.go

```diff
@@ -58,6 +59,7 @@ func NewReleaseCmd(o *ReleaseOptions) *cobra.Command {
 	cmd.Flags().StringVar(&o.outputLocation, "copy-to", defaultArtifactDir, "Output location for artifacts")
 	cmd.Flags().StringVar(&o.repoOutputLocation, "repo-output", "", "Output location for artifacts in repository bundle format")
 	cmd.Flags().BoolVar(&o.debug, "debug", false, "Print verbose debug output")
+	cmd.Flags().StringVarP(&o.tag, "tag", "t", "", "Tag of the image/bundle to be pushed")
```

Comment:
Same refactor of the help message as above

Reply:
Changed it to `cmd.Flags().StringVarP(&o.bundleTag, "tag", "t", "", "Tag pushed with imgpkg bundle (default bundle-<TIMESTAMP>)")`

Reply:
Makes sense, is it not `build-<TIMESTAMP>`?